src/contract/compiler/Base.ts

```diff
@@ -55,6 +55,32 @@ export default abstract class CompilerBase {
     aci: Aci;
   }>;
 
+  /**
+   * Generate contract's ACI by contract's path
+   * Available only in Node.js
+   * @param path - Path to contract source code
+   * @returns ACI
+   */
+  abstract generateAci(path: string): Promise<Aci>;
+
+  /**
+   * Generate contract's ACI by contract's source code
+   * @param sourceCode - Contract source code as string
+   * @param fileSystem - A map of contract filename to the corresponding contract source code to
+   * include into the main contract
+   * @example
+   * ```js
+   * {
+   *   'library.aes': 'namespace TestLib =\n  function sum(x: int, y: int) : int = x + y'
+   * }
+   * ```
+   * @returns ACI
+   */
+  abstract generateAciBySourceCode(
+    sourceCode: string,
+    fileSystem?: Record<string, string>,
```

Comment:
isn't there a type alias for this?

Reply:
I know only the original type `{ [k: string]: string }` 🤷‍♀️ in types generated from swagger file it is defined as `fileSystem?: Record<string, unknown>`

Reply:
ok :)